src/python/nimbusml/base_predictor.py

```diff
@@ -69,13 +66,24 @@ def fit(self, X, y=None, **params):
         set_shape(self, X)
         return self
 
+    @property
+    def _is_fitted(self):
+        """
+        Tells if the predictor was trained.
+        """
+        return (hasattr(self, 'model_') and
+                self.model_ and
+                os.path.isfile(self.model_))
+
     @trace
     def _invoke_inference_method(self, method, X, **params):
         """
         Returns predictions. Can be predicted labels, probabilities
```

Comment:
wrong description

Reply:
This is used for predict, predict_proba, decision_function, and now get_feature_contributions, all of which need a fitted predictor.

Maybe a more accurate description would be "Helper function to return predicted labels, probabilities, decision values, or feature contributions."